libs/application/templates/aosh/work-accident-notification/src/forms/WorkAccidentNotificationForm/AccidentSection/about.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  buildAlertMessageField,
+  buildCustomField,
+  buildDateField,
+  buildDescriptionField,
+  buildHiddenInput,
+  buildMultiField,
+  buildSelectField,
+  buildSubSection,
+  buildTextField,
+  getValueViaPath,
+} from '@island.is/application/core'
+import { accident, sections, shared } from '../../../lib/messages'
+import { NO, YES } from '@island.is/application/types'
+import { MunicipalityDto } from '@island.is/clients/work-accident-ver'
+
+export const aboutSection = buildSubSection({
+  id: 'accident',
+  title: sections.draft.about,
+  children: [
+    buildMultiField({
+      id: 'accidentMultiField',
+      title: accident.about.pageTitle,
+      description: accident.about.description,
+      children: [
+        buildDescriptionField({
+          id: 'accident.description',
+          titleVariant: 'h5',
+          title: accident.about.informationHeading,
+        }),
+        buildDateField({
+          id: 'accident.date',
+          title: accident.about.date,
+          width: 'half',
+          required: true,
+          minDate: new Date('1.1.2020'),
```

Comment:
Should this be dynamic, 4 years back? This will quickly become outdated (no pun intended)

Reply:
This date is defined by VER/AOSH. No request from them to keep this dynamic